pkgs/development/python-modules/textnets/default.nix

```
@@ -78,20 +76,6 @@ buildPythonPackage rec {
     rm -r textnets
   '';
 
```

```suggestion

  __darwinAllowLocalNetworking = true;

```
Can you try adding this please? It might fix the tests on darwin.